src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -107,7 +164,10 @@ public UnorderedCallAssertion MustHaveHappened(Repeated repeatConstraint)
         }
 
         public class ReturnValueConfiguration<TMember>
-            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>, ICallCollectionAndCallMatcherAccessor
+            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>,
+              IAfterCallSpecifiedWithOutAndRefParametersConfiguration<IReturnValueConfiguration<TMember>>,

```

Comment:
I was surprised to see a new interface show up in the list of bases here.
But it's because of [returning this](https://github.com/FakeItEasy/FakeItEasy/pull/804/files/5a163207653185e905712251ee56b0ba7c1732c8#diff-6d9562889008c1f9bfad045e9078aba0R228), no?